ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/ListDetailsActivity.kt

```diff
@@ -26,14 +25,17 @@ class ListDetailsActivity : AppCompatActivity(), ScreenInfoListener {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(R.layout.list_details_activity_main)
         setupToolbar()
         observeSelectedItem()
         observeNavigationStack()
```

Comment:
I think now we should remove observeNavigationStack() in the context of the MainApp since the DisplayHomeAsUpEnabled should always be enabled